### DISCUSSION TITLE
/var/lib/logrotate/pihole is world-readable after upgrade to bullseye

### DISCUSSION THREAD

**Discussion by eddydc** [MichaIng/DietPi#5964] [Q&A]
#### Required Information
cat /boot/dietpi/.version
G_DIETPI_VERSION_CORE=8
G_DIETPI_VERSION_SUB=11
G_DIETPI_VERSION_RC=2
G_GITBRANCH='master'
G_GITOWNER='MichaIng'
G_LIVE_PATCH_STATUS[0]='not applicable'
echo $G_DISTRO_NAME $G_RASPBIAN
bullseye 0
uname -a
`Linux DietPi 5.15.76-v8+ #1597 SMP PREEMPT Fri Nov 4 12:16:41 GMT 2022 aarch64 GNU/Linux`
echo $G_HW_MODEL_NAME
RPi 4 Model B (aarch64)

Upgraded this week two machines to bullseye, by following each instruction in https://dietpi.com/blog/?p=811: an AGH and pihole machine.

All went well expect for the latter where I get the error: state file /var/lib/logrotate/pihole is world-readable and thus can be locked from other unprivileged users. Skipping lock acquisition...

Is this serious and how can this be fixed?

**Comment by Joulinar** [comment]
let's continue discussion on https://github.com/MichaIng/DietPi/issues/5965
